src/Sequence.cc

```diff
@@ -363,6 +364,11 @@ Sequence::execute(Context& context, CommChannel* comm_channel,
         });
 }
 
+std::filesystem::path Sequence::get_folder() const
+{
+    return make_sequence_filename(get_name(), get_unique_id());
```

Comment:
Where is the `make_sequence_filename` function gone? In `serialize_sequence.h/cc` you deleted it...

Reply:
It has become Sequence::get_folder(). 😄